docs/usage.md

```diff
@@ -417,3 +417,75 @@ podman build . \
   --network none \
   --tag atomic-reactor
 ```
+
+### Example: npm
+
+Let's build simple npm project [sample-nodejs-app](https://github.com/cachito-testing/sample-nodejs-app).
+Get the repo if you want to try for yourself:
+
+
+```shell
+git clone https://github.com/cachito-testing/sample-nodejs-app.git
+```
+
+#### Pre-fetch dependencies (npm)
+
+The steps for pre-fetching the dependencies is similar to before, but this time we will use the `npm` package
+manager type. The default behavior path of `.` is assumed.
+
+See [the npm documentation](npm.md) for more details about running Cachi2 for pre-fetching npm dependencies.
+
+```shell
+cachi2 fetch-deps --source ./sample-nodejs-app --output /tmp/cachi2-output '{
```

Comment:
The line breaks are causing an issue if I just try to copy paste the command:
```
Invalid value for 'PKG': Looks like JSON but is not valid JSON: '{\n  "type": "npm",\n}'        
```

Why don't we just leave the example as:
```
cachi2 fetch-deps --source ./sample-nodejs-app --output /tmp/cachi2-output npm
```

Reply:
Nevermind, this was already mentioned in another comment.

Reply:
(btw it's not the line breaks, it's the comma)